colour/volume/macadam_limits.py

```diff
@@ -142,3 +143,197 @@ def is_within_macadam_limits(
     simplex = np.where(simplex >= 0, True, False)
 
     return simplex
+
+
+def macadam_limits(target_brightness, illuminant=()):
```

Comment:
`target_brightness`: Is it really *brightness* here or *luminance*?

The floating-point type in the docstrings hints at the fact that the code is not vectorised and thus a user would need to loop to get the limits for many targets.

We should also have typing annotations here.

`illuminant=()`: This should be better expressed at `illuminant=None` and if it is not passed, we default to *E*.

We should also pass the `CMFS`.

The return value should also be annotated.

The signature should probably be be something along those lines:

```python
macadam_limits(
    luminance: FloatingOrNDArray,
    cmfs: Optional[MultiSpectralDistributions] = None,
    illuminant: Optional[SpectralDistribution] = None,
) -> NDarray:
```

Reply:
Notation with ":" raises errors in "Static Type Checking / macOS-latest - Python 3.9 (pull_request)", only in this test, makes not to much sense. I did it with default definitions like: cmfs = MSDS_CMFS["CIE 1931 2 Degree Standard Observer"]